clients/web/src/lib/components/atoms/Button/Button.svelte

```diff
@@ -0,0 +1,7 @@
+<script lang="ts">
+    export let label:string;
```

Comment:
Lets accept the label via the default slot rather than a prop

See: https://svelte.dev/repl/a295667e0f384c6da13d58f92a24d934?version=3.55.0

Reply:
Done.